File: Encuentro #01/C#/Ejercicio_01/Ejercicio_01/Oyente.cs
using System;

namespace Ejercicio_01
{
    public class Oyente : Participante
    {
        public string Grado_Academico { get; set; }
        public Oyente() { }

        public Oyente(string nombre, string apellido, string email, string grado_Academico) : base(nombre, apellido, email)
        {
            this.Grado_Academico = grado_Academico;
        }
        
        public void LeerOyente()
        {
            LeerParticipante();
            Console.WriteLine("Ingrese el Grado Academico");
            Grado_Academico = Console.ReadLine();
        }

        public void mostrarOyente()
        {
            base.MostrarParticipante();
            Console.WriteLine($"Grado: {Grado_Academico}");
        }
        
    }
}

File: Encuentro #01/Java/Ejercicio 01/src/com/pplam/modelos/Oyente.java
package com.pplam.modelos;
import java.util.Scanner;

public class Oyente extends Participante {
    private String grado;
    public Oyente() {
        super();
    }

    public Oyente(String nombre, String apellido, String email, String grado) {
        super(nombre, apellido, email);
        this.setGrado(grado);
    }

    public void leerOyente() {
        super.leerParticipante();
        System.out.println("Grado: ");
        this.setGrado(new Scanner(System.in).nextLine());
    }

    public void mostrarOyente() {
        super.mostrarParticipante();
        System.out.println("\nGrado:" + this.getGrado());
    }

    public String getGrado() {
        return grado;
    }

    public void setGrado(String grado) {
        this.grado = grado;
    }
}
